ra2ce/analysis/indirect/losses.py

```diff
@@ -548,6 +534,12 @@ def _populate_vehicle_loss_hour(
             raise Exception(
                 f"""{link_type_hazard_range} was not found in the introduced resilience_curve"""
             )
+        
+        if all(ratio <= 1 for ratio_tuple in relevant_curve["functionality_loss_ratio"] for ratio in ratio_tuple):
+                    divisor = 1
+        else:
+            divisor = 100
```

Comment:
For simple `if-else` statements like this you can simply do this to improve readability.
```suggestion
        divisor = 100
        if all(ratio <= 1 for ratio_tuple in relevant_curve["functionality_loss_ratio"] for ratio in ratio_tuple):
                    divisor = 1
```